apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -660,46 +699,67 @@ export const NewHomeScreen = (): ReactElement => {
                       ticketCount={packs.length}
                       cardColor={i % 2 === 0 ? "purple" : "orange"}
                     />
-                    <TicketsContainer
-                      $width={cardWidth}
-                      key={packs.map((pack) => pack.eventTicket.id).join("-")}
-                    >
-                      {packs.map((pack) => {
-                        return (
-                          <CardBody
-                            showDownloadButton={true}
-                            key={pack.eventName}
-                            addOns={
-                              pack.addOns.length > 0
-                                ? {
-                                    text: `View ${pack.addOns.length} add-on items`,
-                                    onClick(): void {
-                                      dispatch({
-                                        type: "set-bottom-modal",
-                                        modal: {
-                                          addOns: pack.addOns,
-                                          modalType: "ticket-add-ons"
-                                        }
-                                      });
+                    <ZappsAndTicketsContainer>
+                      {Object.keys(appConfig.embeddedZapps).length && (
```

Comment:
this should be rendered like we do the arrows, so it won't change by swiping

Reply:
this is a good suggestion but tbh, i don't think this is worth the trouble. all users for devcon will just have one event anyway so no users will notice this feature. if you want to do it and it's fast, feel free to, but i think this is basically low priority.